relay-log/src/setup.rs

```diff
@@ -176,6 +181,7 @@ pub fn init(config: &LogConfig, sentry: &SentryConfig) {
             in_app_include: vec!["relay"],
             release: Some(RELEASE.into()),
             attach_stacktrace: config.enable_backtraces,
+            environment: sentry.environment.clone(),
```

Comment:
Probably more a question for the SDK, but what happens when both `SENTRY_ENVIRONMENT` and `environment` are set?

Reply:
This will behave like `sentry-rust` SDK behaves,  and [according to this part](https://github.com/getsentry/sentry-rust/blob/5be18e5b3a5379b9ca45577bef1c814591ef3c87/sentry/src/defaults.rs#L92) the env will be used when there is nothing set in the config options.